lib/views/list-item/index.js

```diff
@@ -207,9 +227,7 @@ class ListItem extends React.Component {
     debug('gesture end', vx, exceededThreshold);
 
     // fast swipes fling the tile away
-    if (exceededThreshold) {
-      this.swipeAway(vx);
-    }
+    if (exceededThreshold) this.swipeAway(vx);
```

Comment:
Sowweee 😉  That's the current style in this project, if we want to change it, it should be changed everywhere. I could compromise with `if (foo) { return bar; }`, but using three lines when you could use one seems like  unnecessary clutter.

We're lean now, so stuff like this shouldn't matter at this stage 😛